src/fluree/db/ledger/consensus/raft.clj

```diff
@@ -32,10 +31,11 @@
   If multiple parts are returned, additional requests for each part will be
   requested. A snapshot should be broken into multiple parts if it is larger than
   the amount of data you want to push across the network at once."
-  [{:keys [path storage-read] :as config}]
-  (fn [id part]
+  [{:keys [path storage-read]}]
+  (fn [id _]
     ;; TODO: Actually use the part arg or get rid of it - WSM 2020-09-01
     ;; in this example we do everything in one part, regardless of snapshot size
+    ;; part is being passed in as second item to function
```

Comment:
Removed the to-do comment. Since I think that this is a good idea, to handle bundling on network packets based on a configurable parameter, I created FC-1171 to track the enhancement.